res/Rnw/10-CaricamentoFile.Rnw

```diff
@@ -0,0 +1,117 @@
+\end{itemize}
+Per ogniuna di queste estensioni ci sono varie modalità di lettura dei dati e 
+noi ne vedremo una per ciascuno di questi. R mette a disposizione il comando 
+\texttt{read.table} che è un comando generico per laggere dati da file.
+
+\subsection{Caricamento da txt}
+Per caricare i dati da un file con estensione \textbf{txt} è possibile utilizzare
+il comando \texttt{read.delim} al quale è necessario specificare alcune opzioni:
+\begin{itemize}
+  \item il nome del file(se il file è nella stessa cartella da cui lanciamo R)
+  oppure il percorso del file, specificato tra virgolette;
+  \item \texttt{header} da specificare a \texttt{TRUE} oppure \texttt{FALSE} a 
+  seconda che si voglia che la prima riga venga interpretata come intestazione;
+  \item \texttt{sep} per indicare il separatore tra i dati (tra virgolette);
+  \item \texttt{dec} per impostare il carattere utilizzato per dividere le 
+  cifre decimali.
+\end{itemize}
+A questo comando possono essere specificate anche altre opzioni, qui sono 
+riportate le principale. Per una lista esaustiva è sufficiente dare il comando
+\texttt{?read.delim}.
+
+<<eval=FALSE>>=
+read.delim("file.txt", header = TRUE, sep = "\t", dec = ",")
+# In questo esempio si vuole caricare il file 'file.txt', 
+# specificando che nella prima riga troviamo le intestazioni 
+# della tabella, i vari campi sono divisi tramite 'tab' e i 
+# decimali sono specificati tramite la virgola, come nella 
+# notazione italiana
+@
+
+\subsection{Caricamento da csv}
+Il caricamento dei dati da file \textbf{csv} procede in maniera analoga al 
+precedente, l'unica differenza è che il comando da utilizzare questa volta è
+\texttt{read.csv}. Le opzioni da specificare successivamente sono le stesse
+del caso di file con estensione \emph{txt}. Nel caso in cui non venga 
+specificato alcun separatore quello di default utilizzato è la virgola.
+<<eval=FALSE>>=
+read.csv("file.csv", header = TRUE)
+# In questo esempio si vuole caricare il file 'file.csv',
+# specificando che nella prima riga troviamo le intestazioni 
+# della tabella
+@
+
+\subsection{Caricamento da xls}
+Per accedere ai dati salvati in formato \textbf{xls} (il formato di Excel 
+fino al 2007) è necessario importare una libreria chiamata \texttt{gdata}
+tramite il comando \texttt{library(gdata)} (nel caso non fosse installata
+è possibile aggiungerla alle librerie disponibili in locale tramite 
+\texttt{install.packages('gdata')}). A questo punto è possibile dare il 
+comando \texttt{read.xls}, al quale è possibile specificare i seguenti:
+\begin{itemize}
+  \item il nome del file(se il file è nella stessa cartella da cui lanciamo R)
+  oppure il percorso del file, specificato tra virgolette;
+  \item \texttt{sheet} per specificare il nome o il numero del foglio che 
+  vogliamo caricare.
+\end{itemize}
+Anche in questo caso è possibile specificare altre opzione che però noi non 
+vedremo. È inoltre possibile specificare alcune delle opzioni disponibili per 
```

Comment:
`opzioni` viene ripetuto in 3 righe. Se trovi un sinonimo diventa più leggibile il tutto! :blush: